assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/values.yaml

```diff
@@ -0,0 +1,40 @@
+certificates:
+  caKey:
+  caCert:
+
+name: webhook-admission
+
+namespace: kube-system
+
+labels:
+  helm.sh/chart: webhook-admission-0.1.0
+  app.kubernetes.io/instance: webhook-admission
```

Comment:
Aren't those labels automatically added by helm?

Reply:
Yes, it was generated by helpers.tpl file, after removing it, had to use it manually